tensorflow_datasets/image/imagenette_test.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for imagenette: a subset of 10 easily classified classes from
+Imagenet (tench, English springer, cassette player, chain saw, church,
+French horn, garbage truck, gas pump, golf ball, parachute)."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.image import imagenette
+
+
+class ImagenetteTest(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = imagenette.Imagenette
+  SPLITS = {
+      "train": 4,  # Number of fake train example
```

Comment:
your test images have fake random data (good) - but this also means that they don't really compress well (and some take over 3MB).

You can use function from testing/fake_data_utils.py 
(I see that png one is doing the "smart" thing, but the jpeg one is still generating large random images - we should fix that).

Reply:
I have used the function from testing/fake_data_utils.py to generate fake random data. The images are large cuz the size I choose was large something like 2304 x 1728. The only reason for choosing a large size because I have seen similar size images in the original dataset.